cmd/utils/flags.go

```diff
@@ -289,6 +289,11 @@ var (
 		Usage:    "Manually specify optimism",
 		Category: flags.EthCategory,
 	}
+	OverrideL1ArchiveNodeRPC = &cli.StringFlag{
+		Name:     "override.l1archivenoderpc",
+		Usage:    "Manually specify the L1 Archive Node RPC URL, overriding the bundled setting",
```

Comment:
We should make this required so that the software doesn't start without it, otherwise there will be a fork chain that people commonly end up on